### PR TITLE
Fix GCS e2e test

### PR DESCRIPTION
**Pull request by itsankit-google**

### CODE REVIEW

File: src/e2e-test/features/gcs/source/GCSSourceError.feature [hunk 1]
```diff
@@ -35,19 +35,6 @@ Feature: GCS source - Verify GCS Source plugin error scenarios
     Then Enter GCS source property path field "gcsInvalidPathField"
     Then Verify Output Path field Error Message for incorrect path field "gcsInvalidPathField"
 
-  @GCS_DELIMITED_TEST
-  Scenario: To verify Error for incorrect delimiter
```
Comment (sumitjnn) [inline src/e2e-test/features/gcs/source/GCSSourceError.feature:39]: Is this no longer relevant?  Do you know if the behaviour has changed recently?

File: src/e2e-test/features/gcs/source/GCSSourceError.feature [hunk 2]
```diff
@@ -74,14 +61,5 @@ Feature: GCS source - Verify GCS Source plugin error scenarios
...
     Then Toggle GCS source property skip header to true
-    Then Validate output schema with expectedSchema "gcsCsvFileSchema"
-    Then Validate "GCS" plugin properties
-    Then Close the GCS properties
-    Then Open BigQuery sink properties
-    Then Override Service account details if set in environment variables
-    Then Enter the BigQuery sink mandatory properties
-    Then Validate "BigQuery" plugin properties
-    Then Close the BigQuery properties
-    Then Save the pipeline
```
Comment (sumitjnn) [inline src/e2e-test/features/gcs/source/GCSSourceError.feature:85]: Similar question as above

File: src/e2e-test/features/gcs/source/GCSSourceError.feature (hunk 1 above)
Comment (itsankit-google) [reply src/e2e-test/features/gcs/source/GCSSourceError.feature:39]: yeah there were fixes in the plugin, it no longer emits a validation failure, hence we removed it from the develop branch sometime back.

File: src/e2e-test/features/gcs/source/GCSSourceError.feature (hunk 2 above)
Comment (itsankit-google) [reply src/e2e-test/features/gcs/source/GCSSourceError.feature:85]: yeah, now the plugin fails in validation rather than going to the preview stage.

**Review by sumitjnn — APPROVED** [review]
Thanks!

**Pull request merged**